Plaid/Classes/Plaid.m

```diff
@@ -217,6 +219,11 @@ - (void)addLinkUserForProduct:(PlaidProduct)product
   if (pin) {
     parameters[@"pin"] = pin;
   }
+  if (accountFlag) {
```

Comment:
Instead I have added "include_accounts" to dictionary using the boolean values of account flag.
So how you are saying there are no changes needed in current setup. Since " @"include_accounts": @(NO)," needs to be removed since this comes under undocumented parameter.